### DISCUSSION TITLE
Medical support options

### DISCUSSION THREAD

**Discussion by fight-against-hate** [fight-against-hate/fight-against-hate#3] [Q&A]
What doctors or programs are available to ensure medical support -- particularly for minors -- is not interrupted?

**Comment by ghost** [comment]
[FOLX Health](https://www.folxhealth.com) can currently do telehealth for 15 states, but does not support anyone under 18.

**Reply by ghost** [reply]
[Plume](https://getplume.co/map/#hfaq-post-51870) provides estrogen-based HRT to 33 states, testosterone-based to 28 states.  But only serves people age 18-65 (or 19 in Nebraska).

**Comment by ghost** [comment]
A directory of trans-affirming care in the southern states: https://southernequality.org/resources/transinthesouth/

**Reply by ghost** [reply]
NY: https://www1.nyc.gov/site/doh/health/health-topics/transgender-resources.page

NY, NJ, CT: https://qclinic.org/

**Reply by ghost** [reply]
MA: https://www.transhealth.org/our-services/
